packages/sku/src/services/vite/helpers/createConfig.ts

```diff
@@ -9,6 +9,7 @@ import type { SkuContext } from '@/context/createSkuContext.js';
 import skuVitePreloadPlugin from '../plugins/skuVitePreloadPlugin/skuVitePreloadPlugin.js';
 import { fixViteVanillaExtractDepScanPlugin } from '@/services/vite/plugins/esbuild/fixViteVanillaExtractDepScanPlugin.js';
 import { outDir, renderEntryChunkName } from './bundleConfig.js';
+import tsconfigPaths from 'vite-tsconfig-paths';
```

Comment:
`vite-tsconfig-paths` should be added as a dep within `sku`. It's currently only present as a monorepo (top-level) dep.